sequence_classification/tests/test_results_presenter.py

```diff
@@ -0,0 +1,79 @@
+from sequence_classification.results_presenter import ResultsPresenter
+import pytest
+import numpy as np
+from scipy.stats import friedmanchisquare
+
+@pytest.fixture
+def prepare_data():
+    data = {'small_imdb':
+             {'SVM': [{'conf_matrix_train': np.array([[370,   0],
```

Comment:
Nieczytelny format danych. Porponuję coś takiego:
```Python
{'SVM': [
    {'conf_matrix_train': np.array([[370,   0], [0, 380]]),
     'conf_matrix_test': np.array([[100,  36], [22,  92]]),
     'params': {}},
    {'conf_matrix_train': np.array([[382,   0], [0, 368]]),
     'conf_matrix_test': np.array([[98,  26], [17, 109]]),
     'params': {}}
 ],
}
```

Reply:
Dodatkowo elementy "conf_matrix_train" są niepotrzebne, więc też by można wywalić je, bo by to zajmowało 2 razy mniej miejsca